trezorlib/tests/device_tests/test_msg_stellar_sign_transaction.py

```diff
@@ -56,10 +85,71 @@ def test_sign_tx_account_merge_op(self):
+
+        op = proto.StellarCreateAccountOp()
+        op.new_account = stellar.address_to_public_key("GBOVKZBEM2YYLOCDCUXJ4IMRKHN4LCJAE7WEAEA2KF562XFAGDBOB64V")
+        op.starting_balance = 1000333000
+
+        tx = self._create_msg()
+
+        response = self.client.stellar_sign_transaction(tx, [op], self.get_address_n(), self.get_network_passphrase())
+
+        assert b64encode(response.signature) == b'vrRYqkM4b54NrDR05UrW7ZHU7CNcidV0fn+bk9dqOW1bCbmX3YfeRbk2Tf1aea8nr9SD0sfBhtrDpdyxUenjBw=='
+
+    def test_sign_tx_payment_op_native(self):
+        """Native payment of 50.0111 XLM to GBOVKZBEM2YYLOCDCUXJ4IMRKHN4LCJAE7WEAEA2KF562XFAGDBOB64V"""
+        self.setup_mnemonic_nopin_nopassphrase()
+
+        op = proto.StellarPaymentOp()
+        op.amount = 500111000
+        op.destination_account = stellar.address_to_public_key("GBOVKZBEM2YYLOCDCUXJ4IMRKHN4LCJAE7WEAEA2KF562XFAGDBOB64V")
+
+        tx = self._create_msg()
+
+        response = self.client.stellar_sign_transaction(tx, [op], self.get_address_n(),self.get_network_passphrase())
+
+        assert b64encode(response.signature) == b'pDc6ghKCLNoYbt3h4eBw+533237m0BB0Jp/d/TxJCA83mF3o5Fr4l5vwAWBR62hdTWAP9MhVluY0cd5i54UwDg=='
+
+    def test_sign_tx_payment_op_custom_asset1(self):
+        """Custom asset payment (code length 1) of 50.0111 X to GBOVKZBEM2YYLOCDCUXJ4IMRKHN4LCJAE7WEAEA2KF562XFAGDBOB64V"""
+        self.setup_mnemonic_nopin_nopassphrase()
+
+        op = proto.StellarPaymentOp()
+        op.amount = 500111000
+        op.destination_account = stellar.address_to_public_key("GBOVKZBEM2YYLOCDCUXJ4IMRKHN4LCJAE7WEAEA2KF562XFAGDBOB64V")
+
+        op.asset = proto.StellarAssetType(1, "X", stellar.address_to_public_key("GAUYJFQCYIHFQNS7CI6BFWD2DSSFKDIQZUQ3BLQODDKE4PSW7VVBKENC"))
+
+        tx = self._create_msg()
+
+        response = self.client.stellar_sign_transaction(tx, [op], self.get_address_n(), self.get_network_passphrase())
+
+        assert b64encode(response.signature) == b'ArZydOtXU2whoRuSjJLFIWPSIsq3AbsncJZ+THF24CRSriVWw5Fy/dHrDlUOu4fzU28I6osDMeI39aWezg5tDw=='
+
+    def test_sign_tx_payment_op_custom_asset12(self):
+        """Custom asset payment (code length 12)of 50.0111 X to GBOVKZBEM2YYLOCDCUXJ4IMRKHN4LCJAE7WEAEA2KF562XFAGDBOB64V"""
+        self.setup_mnemonic_nopin_nopassphrase()
+
+        op = proto.StellarPaymentOp()
+        op.amount = 500111000
+        op.destination_account = stellar.address_to_public_key("GBOVKZBEM2YYLOCDCUXJ4IMRKHN4LCJAE7WEAEA2KF562XFAGDBOB64V")
+
+        op.asset = proto.StellarAssetType(2, "ABCDEFGHIJKL", stellar.address_to_public_key("GAUYJFQCYIHFQNS7CI6BFWD2DSSFKDIQZUQ3BLQODDKE4PSW7VVBKENC"))
+
+        tx = self._create_msg()
+
+        response = self.client.stellar_sign_transaction(tx, [op], self.get_address_n(), self.get_network_passphrase())
+
+        assert b64encode(response.signature) == b'QZIP4XKPfe4OpZtuJiyrMZBX9YBzvGpHGcngdgFfHn2kcdONreF384/pCF80xfEnGm8grKaoOnUEKxqcMKvxAA=='
+
```

Comment:
Extra line, there should be only one (that's why the tests are failing).
`./trezorlib/tests/device_tests/test_msg_stellar_sign_transaction.py:154:5: E303 too many blank lines (2)`